### PR TITLE
Fixed workflow check-atlantis

### PR DESCRIPTION
**Pull request by tponomarenko**

### CODE REVIEW

File: .idea/.gitignore
Comment (romcheg) [inline .idea/.gitignore:1]: Please create .gitignore in the root catalogue.

You can use Github's recommended template for it: https://github.com/github/gitignore/blob/main/Terraform.gitignore

You need to delete and commit .idea folder before adding it to .gitignore.

Comment (tponomarenko) [reply .idea/.gitignore:1]: done

File: .github/workflows/check_atlantis.yaml [hunk 1]
```diff
@@ -21,12 +7,12 @@ jobs:
...
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Check Docs for all modules
         uses: terraform-docs/gh-actions@v1.2.0
         with:
           find-dir: .
           config-file: .terraform-docs.yml
-          output-file: README.md
```
Comment (romcheg) [inline .github/workflows/check_atlantis.yaml:30]: does it mean the default value is README.md?

File: .github/workflows/check_atlantis.yaml [hunk 2]
```diff
@@ -21,12 +7,12 @@ jobs:
...
+          output-method: ""
+          template: ""
           fail-on-diff: true
-          git-push: false
```
Comment (romcheg) [inline .github/workflows/check_atlantis.yaml:32]: same question as on L30

File: .github/workflows/check_atlantis.yaml [hunk 3]
```diff
@@ -1,17 +1,3 @@
-# Copyright 2024 Hauki Tech sp. z o.o.
```
Comment (romcheg) [inline .github/workflows/check_atlantis.yaml:1]: Please adjust the license header to the previously agreed format:

```
Copyright [yyyy] [name of copyright owner]
SPDX-License-Identifier: Apache-2.0
```

File: .github/workflows/check_atlantis.yaml (hunk 1 above)
Comment (tponomarenko) [reply .github/workflows/check_atlantis.yaml:30]: yes

File: .github/workflows/check_atlantis.yaml (hunk 2 above)
Comment (tponomarenko) [reply .github/workflows/check_atlantis.yaml:32]: yes

File: .github/workflows/check_atlantis.yaml (hunk 3 above)
Comment (tponomarenko) [reply .github/workflows/check_atlantis.yaml:1]: done

**Review by romcheg — APPROVED** [review]

**Pull request merged**